experiment/nursery/create_bazel_cache_rcs.sh

```diff
@@ -20,12 +20,24 @@
 CACHE_HOST="http://bazel-cache.default:8080"
 
 make_bazel_rc () {
+    # this is the default for recent releases but we set it explicitly
+    # since this is the only hash our cache supports
     echo "startup --host_jvm_args=-Dbazel.DigestFunction=sha256" >> $1
+    # caching args for build
+    # - use remote strategy for all the things
     echo "build --spawn_strategy=remote" >> $1
     echo "build --strategy=Javac=remote" >> $1
     echo "build --genrule_strategy=remote" >> $1
-    echo "build --remote_rest_cache=${CACHE_HOST}" >> $1
+    # - use the http cache ...
+    echo "build --remote_http_cache=${CACHE_HOST}" >> $1
+    # - don't fail if the cache is unavailable
     echo "build --remote_local_fallback" >> $1
+    # same args for, but for test
+    echo "test --spawn_strategy=remote" >> $1
+    echo "test --strategy=Javac=remote" >> $1
+    echo "test --genrule_strategy=remote" >> $1
+    echo "test --remote_http_cache=${CACHE_HOST}" >> $1
+    echo "test --remote_local_fallback" >> $1
```

Comment:
AFAICT from the manual these flags weren't applied to `bazel test` previously. I think this should be enough.

Reply:
are you sure? I'm pretty certain that test inherits from build.